tests/integration/components/user-skills-input-test.js

```diff
@@ -20,12 +19,8 @@ let mockStore = {
 };
 
 let mockUserSkillsService = {
-  findUserSkill() {
-    return K;
-  },
-  removeSkill() {
-    return K;
-  }
+  findUserSkill() {},
+  removeSkill() {}
```

Comment:
For these methods, I believe the component expects something to be returned. You may just want to `return true` or something similar. Should help with a couple of the failing tests.